src/github/client/github-client-interceptors.ts

```diff
@@ -104,7 +105,7 @@ export const handleFailedRequest = (logger: Logger) =>
 
 		if (response) {
 			const status = response?.status;
-			const errorMessage = `Error executing Axios Request ` + error.message;
+			const errorMessage = `Error executing Axios Request (${toUpper(config.method)} ${config.baseURL}${config.url}): ` + error.message;
```

Comment:
At one point when I was investigating the 501 that Axios always returned I added some extra information to the error logs so I didn't have to guess which call went wrong.